src/main/java/com/allocadia/carbonite/CarbonitePersist.java

```diff
@@ -0,0 +1,54 @@
+package com.allocadia.carbonite;
+
+import org.springframework.jdbc.core.support.JdbcDaoSupport;
+
+import com.allocadia.carbonite.utils.QueryUtils;
+import com.google.common.base.Joiner;
+
+import java.lang.reflect.Field;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Map;
+
+import lombok.SneakyThrows;
+
+public class CarbonitePersist<T extends PersistedObject> extends JdbcDaoSupport {
+
+    public Collection<T> save(Carbonite carbonite, Collection<T> objects) {
+        ModifiedObjects modifiedObjects = ModifiedObjects.from(objects);
+        Class<?> clazz = objects.iterator().next().getClass();
+        PersistenceInfo<PersistedObject> info = (PersistenceInfo<PersistedObject>) carbonite.getPersistenceInfo(clazz);
+        
+        persistNew(modifiedObjects.getNewObjects(), info);
+        //TODO: persistDirty
+    }
+
+    @SneakyThrows
+    private void persistNew(List<PersistedObject> newObjects, PersistenceInfo<PersistedObject> info) {
+        Map<Field, String> field2Column = info.getField2Column();
+        String baseSql = Joiner.on("\n").join(
+            "INSERT INTO " + info.getTableName(),
+            "(" + String.join(", ", field2Column.values()) + ")",
```

Comment:
Should we only specify the fields which are dirty to allow the other fields to use DB defaults?